common/cultures/31_pony.txt

```diff
@@ -3,32 +3,32 @@ equestrian	= {
 
 	alicorn = {
 		primary = EQU
+		dynasty_names = {
+			"of The Alicorns" "of The Earth Ponies" "of The Pegasi" "of The Unicorns"
```

Comment:
of Alicorns sounds off. of The Alicorns, now we know what we are talking about! The same goes for the other dynasty names